core/src/mill/eval/Evaluator.scala

```diff
@@ -30,7 +30,6 @@ case class Labelled[T](task: NamedTask[T],
 }
 case class Evaluator[T](home: Path,
                         outPath: Path,
-                        externalOutPath: Path,
```

Comment:
this parameter was always equal to `outPath` (except in one test), so I took the liberty to remove. I think I could add a `external-modules` segment in out in the new `destSegments` conditioned by the module/task being "external". Thoughts ?

Reply:
We re-used externalOutPaths in tests to speed up the test suite; otherwise re-initializing the Zinc incremental compiler made them take forever. Do you have a good reason to believe this is no longer the case?